wrappers/SMath/coolprop_wrapper/Functions/CoolProp_HAProps.cs

```diff
@@ -3,136 +3,136 @@
-                        "Output The output parameter, one of \"T\",\"D\",\"H\",etc.\r\n" +
-                        "Name1 The first state variable name, one of \"T\",\"D\",\"H\",etc.\r\n" +
-                        "Prop1 The first state variable value\r\n" +
-                        "Name2 The second state variable name, one of \"T\",\"D\",\"H\",etc.\r\n" +
-                        "Prop2 The second state variable value\r\n" +
-                        "Name3 The third state variable name, one of \"T\",\"D\",\"H\",etc.\r\n" +
-                        "Prop3 The third state variable value";
+        TermInfo IFunction.GetTermInfo(string lang)
+        {
+            string funcInfo = "(Output, Name1, Prop1, Name2, Prop2, Name3, Prop3) Return a humid air property\r\n" +
+                "Output: The output parameter, one of \"T\", \"D\", \"H\", etc...\r\n" +
+                "Name1: The first state variable name, one of \"T\", \"D\", \"H\", etc...\r\n" +
+                "Prop1: The first state variable value\r\n" +
+                "Name2: The second state variable name, one of \"T\", \"D\", \"H\", etc...\r\n" +
+                "Prop2: The second state variable value\r\n" +
+                "Name3: The third state variable name, one of \"T\", \"D\", \"H\", etc...\r\n" +
+                "Prop3: The third state variable value";
 
-      var argsInfos = new [] {
-        new ArgumentInfo(ArgumentSections.String),
-        new ArgumentInfo(ArgumentSections.String),
-        new ArgumentInfo(ArgumentSections.RealNumber),
-        new ArgumentInfo(ArgumentSections.String),
-        new ArgumentInfo(ArgumentSections.RealNumber),
-        new ArgumentInfo(ArgumentSections.String),
-        new ArgumentInfo(ArgumentSections.RealNumber),
-      };
+            var argsInfos = new [] {
+                new ArgumentInfo(ArgumentSections.String),
+                new ArgumentInfo(ArgumentSections.String),
+                new ArgumentInfo(ArgumentSections.RealNumber),
+                new ArgumentInfo(ArgumentSections.String),
+                new ArgumentInfo(ArgumentSections.RealNumber),
+                new ArgumentInfo(ArgumentSections.String),
+                new ArgumentInfo(ArgumentSections.RealNumber),
+            };
 
-      return new TermInfo(inf.Text,
-                          inf.Type,
-                          funcInfo,
-                          FunctionSections.Unknown,
-                          true,
-                          argsInfos);
-    }
+            return new TermInfo(inf.Text,
+                                inf.Type,
+                                funcInfo,
+                                FunctionSections.Unknown,
+                                true,
+                                argsInfos);
+        }
 
-    bool IFunction.ExpressionEvaluation(Term root, Term[][] args, ref SMath.Math.Store context, ref Term[] result)
```

Comment:
not sure if this is the only reason behind this change, but context is a reference type, thus not using ref should be an issue only if one wants to initialize a new instance (that shouldn't happens plugin-side);